### PR TITLE
Fix/tao 7637 eslint errors for droppable.js

### PR DESCRIPTION
**Pull request by rkarolis**
related to https://oat-sa.atlassian.net/browse/TAO-7637
Fix eslint warnings and errors

Test if droppable.js eslint standards in code applied correctly.
In Item Authoring check if dropping and moving new item the placeholder works correctly and it is removed while moving away from item page.

### CODE REVIEW

File: views/js/qtiCreator/editor/gridEditor/droppable.js [hunk 1]
```diff
@@ -220,9 +237,12 @@ define([
         //bind all event handlers:
         $el.on('mouseenter.gridEdit.gridDragDrop', '[class^="col-"]:not(.new-col), [class*=" col-"]:not(.new-col)', _.debounce(function(e){
```
Comment (ricproenca-tao) [inline views/js/qtiCreator/editor/gridEditor/droppable.js]: Just a suggestion... 
This line is too long.
To improve readability you can  put `_.debounce(function(e){` on a new line

File: views/js/qtiCreator/editor/gridEditor/droppable.js [hunk 2]
```diff
@@ -28,18 +28,28 @@ define([
...
+        var data = options.data || {};
 
-        var $placeholder = $('<div>', {'id' : 'qti-block-element-placeholder', 'class' : 'qti-droppable-block-hover'}),
-            marginWidth = parseFloat($el.find('[class^="col-"]:last, [class*=" col-"]:last').css('margin-left')),
-            isEmpty = ($el.children('.grid-row').length === 0);
+        var $placeholder = $('<div>', {'id' : 'qti-block-element-placeholder', 'class' : 'qti-droppable-block-hover'});
+        var marginWidth = parseFloat($el.find('[class^="col-"]:last, [class*=" col-"]:last').css('margin-left'));
+        var isEmpty = ($el.children('.grid-row').length === 0);
+
+        var _appendPlaceholder;
```
Comment (ricproenca-tao) [inline views/js/qtiCreator/editor/gridEditor/droppable.js]: I don't think using this type of var is the TAO way. `_appendPlaceholder`.
There are a lot in this file.
 Can you change it please?

File: views/js/qtiCreator/editor/gridEditor/droppable.js [hunk 3]
```diff
@@ -174,28 +190,29 @@ define([
...
 
         //recalculate the cols height according to new layout
-        var _resetColsHeight = function($col, self){
+        _resetColsHeight = function($col, asSelf){
 
             var maxHeight = 0;
-            if(self === undefined){
-                self = true;
+            var $cols;
+            if(!asSelf){
```
Comment (ricproenca-tao) [inline views/js/qtiCreator/editor/gridEditor/droppable.js]: I don't understand the usage of `asSelf` in this function.
I think that is possible to improve that.
What do you think?

File: views/js/qtiCreator/editor/gridEditor/droppable.js (hunk 1 above)
Comment (rkarolis) [reply views/js/qtiCreator/editor/gridEditor/droppable.js]: Developers are still using the dash to defined that variable is used privately.

File: views/js/qtiCreator/editor/gridEditor/droppable.js (hunk 2 above)
Comment (rkarolis) [reply views/js/qtiCreator/editor/gridEditor/droppable.js]: Developers are still using the dash to defined that variable is used privately.

File: views/js/qtiCreator/editor/gridEditor/droppable.js (hunk 3 above)
Comment (rkarolis) [reply views/js/qtiCreator/editor/gridEditor/droppable.js]: good catch!

**Pull request merged**